manim/mobject/types/vectorized_mobject.py

```diff
@@ -983,13 +983,13 @@ def point_from_proportion(self, alpha: float) -> np.ndarray:
             Point evaluated.
         """
 
-        curves_with_lengths = list(self.get_curve_functions_with_lengths())
+        if alpha >= 1:
+            return self.points[-1]
```

Comment:
This doesn't make sense to me. #1302 will make alpha values less than 0 and greater than 1 raise an error, which I think is appropriate

Reply:
> Also these changes will need to also be passed onto OpenGLVMobject. Currently we're in a wonky in-between state as we transition to OpenGL rendering which requires code duplication.

I didn't work with the openGL version until now, so I will not include that in this PR.

Reply:
Changed the `>=1` to `==1`. I still think this is a reasonable check, since this case appears quite often in animations.